docs/manuals/source/Appendix/DisasterRecoveryUsingBareos.rst

```diff
@@ -128,19 +158,339 @@ to create the rescue image. If you used the configuration example above, you wil
+After restoring the files, ReaR restored the bootloader.
+The recovery is now completed and the system can be rebooted.
+
+The ReaR Bareos backend also works in ReaRs **Automatic Recover** mode,
+selectable at boot,
+which recreates the system without manual interaction.
+
+
+Advanced ReaR Bareos Backend Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _section-rear-snapshot-package-installation:
+
+Installing a ReaR Snapshot Package
+''''''''''''''''''''''''''''''''''
+
+The current (2024) stable release of ReaR is 2.7,
+which does not support Bareos > 22.
+Therefore until a newer version is released,
+a ReaR snapshot package must be used:
+
+* Download the ZIP file that best matches your distribution from https://github.com/rear/rear/releases/tag/snapshot
+* Unpack the ZIP file and install the package on the client.
+
+For details, refer to https://relax-and-recover.org/download/
+
+
+.. _section-named-console-for-rear:
+
+Configure a Named Console for ReaR
+''''''''''''''''''''''''''''''''''
+
+The default :command:`bconsole` configuration on the |dir| offers full access to the |dir|.
+This is normally not wanted from a client systems.
+
+Instead you configure a :config:option:`dir/console` for every Bareos client using ReaR,
+limiting the access to only one system and the required console commands:
+
+.. code-block:: bareosconfig
+   :caption: :file:`bareos-dir.d/console/bareosclient-console.conf`
+
+   Console {
+      # individual per client
+      Name = "bareosclient-console"
+      Password = "secret"
+      Client ACL = "bareosclient-fd"
+
+      # identical for all clients
+      Description = "Restricted console used by ReaR"
+      Command ACL = "."
+      Command ACL = ".api"
+      Command ACL = ".client"
+      Command ACL = ".clients"
+      Command ACL = ".fileset"
+      Command ACL = ".filesets"
+      Command ACL = ".help"
+      Command ACL = ".jobs"
+      Command ACL = ".jobstatus"
+      Command ACL = ".status"
+      Command ACL = "exit"
+      Command ACL = "help"
+      Command ACL = "list"
+      Command ACL = "llist"
+      Command ACL = "restore"
+      Command ACL = "show"
+      Command ACL = "status"
+      Command ACL = "version"
+      Command ACL = "wait"
+      Catalog ACL = *all*
+      FileSet ACL = *all*
+      Job ACL = *all*
+      Plugin Options ACL = *all*
+      Pool ACL = *all*
+      Schedule ACL = *all*
+      Storage ACL = *all*
+      Where ACL = *all*
+   }
+
+Create the file on the |dir| and :bcommand:`reload` the configuration.
+Of course a individual password should be used.
+
+The corresponding :file:`/etc/bareos/bconsole.conf` on the ReaR client will look like this.
+
+.. code-block:: bareosconfig
+   :caption: :file:`/etc/bareos/bconsole.conf`
+
+   Director {
+      Name = "bareos-dir"
+      Address = "bareos.example.com"
+      Password = ""
+   }
 
-   root@host:~# rear recover
+   Console {
+      Name = "bareosclient-console"
+      Password = "secret"
```

Comment:
Maybe hint that this needs to be protected with filesystem ACL and that ReaR will take it along and that the rescue image should be treated as a credential?

Reply:
done